ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Index.cshtml

```diff
@@ -4,187 +4,187 @@
+                        </a>
+                    </span>
+                    <strong class="govuk-tag @Model.TaskList.GeneralInformationTaskListStatus.CssClass app-task-list__tag" id="general-information-status">
+                        @Model.TaskList.GeneralInformationTaskListStatus.Status
+                    </strong>
+                </li>
+                <li class="app-task-list__item">
+                    <span class="app-task-list__task-name">
+                        <a class="govuk-link" asp-page="@Links.SchoolPerformance.ConfirmSchoolPerformance.Page" asp-route-id="@Model.Project.Id" aria-describedby="school-performance-status">
+                            School performance (Ofsted information)
+                        </a>
+                    </span>
+                    <strong class="govuk-tag govuk-tag--grey app-task-list__tag" id="school-performance-status">Reference only</strong>
+                </li>
+                <li class="app-task-list__item">
+                    <span class="app-task-list__task-name">
+                        <a class="govuk-link" asp-page="@Links.RationaleSection.ConfirmProjectAndTrustRationale.Page" asp-route-id="@Model.Project.Id" aria-describedby="rationale-status">
+                            Rationale
+                        </a>
+                    </span>
+                    <strong class="govuk-tag @Model.TaskList.RationaleTaskListStatus.CssClass app-task-list__tag" id="rationale-status">
+                        @Model.TaskList.RationaleTaskListStatus.Status
+                    </strong>
+                </li>
+                <li class="app-task-list__item">
+                    <span class="app-task-list__task-name">
+                        <a class="govuk-link" asp-page="@Links.RisksAndIssuesSection.ConfirmRisksAndIssues.Page" asp-route-id="@Model.Project.Id" aria-describedby="risks-and-issues">
+                            Risks and issues
+                        </a>
+                    </span>
+                    <strong class="govuk-tag @Model.TaskList.RisksAndIssuesTaskListStatus.CssClass app-task-list__tag" id="risks-and-issues-status">
+                        @Model.TaskList.RisksAndIssuesTaskListStatus.Status
+                    </strong>
+                </li>
+                <li class="app-task-list__item">
+                    <span class="app-task-list__task-name">
+                        <a class="govuk-link" asp-page="@Links.LegalRequirements.Summary.Page" asp-route-id="@Model.Project.Id" aria-describedby="legal-requirements" data-cy="@Select.TaskList.Links.LegalRequirements">
+                            Legal requirements
+                        </a>
+                    </span>
+                    <strong class="govuk-tag @Model.TaskList.LegalRequirementsTaskListStatus.CssClass app-task-list__tag" id="legal-status" @Select.TaskList.LegalRequirements.Status.ToData("cy")>
+                        @Model.TaskList.LegalRequirementsTaskListStatus.Status
```

Comment:
Can you put back the .ToDescription()
Just so it displays the description rather than the enum name.

![image](https://user-images.githubusercontent.com/109524605/190665673-8f2fe082-bfa3-4134-8677-e385745e26ef.png)

Reply:
Done.

Reply:
Updated to be done when the mapping occurs.